release-notes/2024-05/README.md

```diff
@@ -0,0 +1,212 @@
+
+# SystemLink Enterprise release 2024-05 Release Notes
+
+The 2024-05 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Navigate SystemLink Enterprise faster with the intuitively organized UI.
+- Visualize data tables in a data space to gain insights into your time-series data.
+- Analyze test results with Jupyter Notebooks.
+- Annotate steps with keywords from a data space.
+- Save custom column configurations for the product specifications grid.
```

Comment:
We need to add one more item. I'm also working with Vera to add these to her publication:

- Define and apply custom states to a system

Reply:
I've added this new bullet

Reply:
I've also added the bullet to the list in the User Manual. You can see it here: https://ni-staging.zoominsoftware.io/docs/en-US/bundle/systemlink-enterprise/page/systemlink-enterprise-new-features-and-changes.html

Reply:
@CIakab-NI Here are the links to the related topics:
⁠https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/analyzing-test-results-with-jupyter-notebooks.html
⁠https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/visualizing-data-tables-in-a-data-space.html
⁠https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/editing-test-result-keywords-from-a-data-space.html
⁠https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/creating-a-test-plan.html

Reply:
I did check your list and updated mine to be identical.

Thanks for the links @vpstoynova - I did add the links to Github (for some reasone 2x of the links are not highlighted although added to the Readme.md).